### ISSUE TITLE
Heroku deploy error 

### ISSUE THREAD

**Issue by SimonLab** [dwyl/dwylbot#174]
![image](https://user-images.githubusercontent.com/6057298/36139754-b220a98e-1096-11e8-86dd-366f9eacac27.png)

- Why is Heroku running Elixir 1.3?
- How to update to 1.4?

**Comment by SimonLab** [comment]
We might need to update the buildback config:
https://github.com/dwyl/dwylbot/blob/master/elixir_buildpack.config

see:
- https://hexdocs.pm/phoenix/heroku.html
- https://github.com/HashNuke/heroku-buildpack-elixir

**Comment by nelsonic** [comment]
Yeah, the HashNuke buildpack doesn't have the latest Elixir ... 🙄
